reana_server/scheduler.py

```diff
@@ -173,6 +173,14 @@ def on_message(self, workflow_submission, message):
                     exc_info=True,
                 )
                 requeue = False
+            except HTTPConflict as e:
+                logging.error(
+                    f"Workflow failed to start because "
+                    f"of Rabbit MQ message conflict. \n"
```

Comment:
f-strings not needed. Maybe we can make it a one-liner?

Reply:
After changing it I've noticed that CI is not passing. Apparently there was a new [WTForms](https://pypi.org/project/WTForms/3.0.0/) release two days ago which [broke our invenio dependencies](https://github.com/reanahub/reana-server/runs/4152097221?check_suite_focus=true#step:6:23). I had to pin `WTForms` version to fix it